### DISCUSSION TITLE
Kimai Demo down, 500 error

### DISCUSSION THREAD

**Discussion by bigvictorio** [kimai/kimai#4244] [On-Premise: Installation & Updates]
Hi,

I was trying to test something, and this may not be the right communication channel. If @kevinpapst already knows about this then you can delete this thread.

https://demo.kimai.org/ and https://demo-empty.kimai.org/ are down with HTTPS 500 Errors.

**Comment by kevinpapst** [comment]
Thanks, fixed 👍
